### PR TITLE
Added AMC Column in Projects table 

### PR DESCRIPTION
**Pull request by dhruvshrinet1**
Targets #{2368}

### Description
Added is_amc Column in Projects table of Boolean type 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/create.blade.php [hunk 1]
```diff
@@ -12,7 +12,9 @@
                 <div class="card-header">
                     <span>Project Details</span>
                 </div>
+               
                 @include('project::subviews.create-project-details')
+
```
Comment (P4NK4J) [inline Modules/Project/Resources/views/create.blade.php:17]: Let's remove extra spaces from here

File: Modules/Expense/Http/Controllers/RecurringExpenseController.php [hunk 2]
```diff
@@ -31,6 +31,14 @@ public function create()
...
     {
         // ToDo:: we need to add validations here.
+        $validated = $request->validate([
+        'name' => 'required',
+        'frequency' => 'required',
+        'initial_due_date' => 'required',
+        'currency' =>  'required',
+        'amount' => 'required',
+        ]);
+
```
Comment (P4NK4J) [inline Modules/Expense/Http/Controllers/RecurringExpenseController.php:41]: I think these changes are already in an existing PR, can we merge them and make a single PR ?

**Review by P4NK4J — COMMENTED** [review]
left some comments, please check

File: Modules/Project/Resources/views/subviews/create-project-details.blade.php [hunk 3]
```diff
@@ -59,7 +59,12 @@
                     placeholder="Enter Google Chat Webhook URL"
                     value="{{ old('google_chat_webhook_url') }}">
             </div>
-            <div class="form-group offset-md-1 col-md-2" v-if="projectType === 'fixed-budget'">
+            <div class="form-group offset-md-1 col-md-5">
+                <input type="checkbox" id="isamc" name="isamc">
+                <label for="isamc">AMC</label><br>
+
```
Comment (P4NK4J) [inline Modules/Project/Resources/views/subviews/create-project-details.blade.php:65]: Extra space here

**Review by P4NK4J — APPROVED** [review]
Minor comment, else looks good. Please remove the spaces

File: Modules/Project/Resources/views/subviews/create-project-details.blade.php (hunk 3 above)
Comment (dhruvshrinet1) [reply Modules/Project/Resources/views/subviews/create-project-details.blade.php]: spacing adjusted

**Pull request merged**